dtests/src/test/java/io/snappydata/hydra/diskFullTests/diskFullTestWithDeletes.conf

```diff
@@ -0,0 +1,44 @@
+hydra.Prms-testRequirement = "Test to verify the cluster stability while delete operations are being executed during diskfull scenario";
+hydra.Prms-testDescription = "This test starts the snappy cluster.
+    Creates column tables with key columns . After initialization,data is ingested in the cluster using inserts.
+    Then batch delete operations are executed, in parallel to the deletion task, we also execute fallocate command to
+    create files of specific size in order to create the disk full situation.
+    In closeTask, the cluster is shutdown ,the pre created files are cleared, so as to free up disk space.
+    Then the cluster is started and validation is done to check if cluster is up and running and that there is no data loss";
+
+INCLUDE $JTESTS/io/snappydata/hydra/northwind/startEmbeddedModeCluster.conf;
+//create and load tables
+INITTASK    taskClass   = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeSQLScripts
+            io.snappydata.hydra.cluster.SnappyPrms-sqlScriptNames = ${scriptName}
+            threadGroups = snappyThreads
+            ;
+
+TASK        taskClass   = io.snappydata.hydra.cdcConnector.SnappyCDCTest taskMethod  = HydraTask_executeFallocate
+            io.snappydata.hydra.cdcConnector.SnappyCDCPrms-fileSize = ${fileSize}
+            io.snappydata.hydra.cdcConnector.SnappyCDCPrms-snappyFileLoc = ${snappyFileLoc}
+            maxThreads = 1
+            maxTimesToRun = 5
+            startInterval = 5
+            endInterval   = 5
+            threadGroups = locatorThreads
+            ;
+
+TASK        taskClass  = io.snappydata.hydra.cdcConnector.SnappyCDCTest taskMethod  = HydraTask_executeOps
+            io.snappydata.hydra.cdcConnector.SnappyCDCPrms-snappyFileLoc = ${snappyFileLoc}
+            io.snappydata.hydra.cdcConnector.SnappyCDCPrms-dataLocation = ${deleteQueryFilePath}
+            maxThreads = 1
```

Comment:
delete statement will be executed only once, so chances of getting disk full while delete is happening are low. So delete can also have some start time interval.